apps/api/package.json

```diff
@@ -49,9 +49,9 @@
     "pino": "^8.1.0",
     "pino-http": "^8.1.0",
     "pino-pretty": "^8.1.0",
+    "rhea": "3.0.1",
     "swagger-ui-express": "^4.4.0",
-    "xstate": "4.32.1",
-    "rhea": "3.0.1"
+    "xstate": "4.32.1"
```

Comment:
is this right?

Reply:
@lutamKC  I wondered this too - it was the result of me running npm ci. Is this not correct?

Reply:
I will revert this